src/lr_scheduler/linear_lr.rs

```diff
@@ -0,0 +1,40 @@
+use super::Schedule;
+
+pub struct LinearLR {
+    pub total_iters: usize,
+    pub start_factor: f64,
+    pub end_factor: f64,
```

Comment:
Similiar to my other comment, there are some [constraints](https://pytorch.org/docs/stable/_modules/torch/optim/lr_scheduler.html#LinearLR) on what `start_factor` and `end_factor` can be